sources/packages/api/src/helpers/utilfunctions.ts

```diff
@@ -1,11 +1,12 @@
-import { StudentConfirmInfo } from "../route-controllers/student/models/student.dto";
+import { StudentInfo } from "../route-controllers/student/models/student.dto";
 export default class Helper {
-  static mapAddressAttributes(fromObject: any, toObject: StudentConfirmInfo) {
-    toObject.addressLine1 = fromObject.addressLine1;
-    toObject.addressLine2 = fromObject.addressLine2;
-    toObject.city = fromObject.city;
-    toObject.provinceState = fromObject.province;
-    toObject.country = fromObject.country;
-    toObject.postalCode = fromObject.postalCode;
+  static mapAddressAttributes(fromObject: any, toObject: StudentInfo) {
```

Comment:
Is there a particular reason to have a mapper for this?
Seems to me that the only property that is different between both objects is "province".
If that is the case, I would recommend not create a helper only for this case.
If there is a strong reason to keep it, please change the "fromObject" to a specific type.

Reply:
Got rid of the mapper